parse.go

```diff
@@ -343,7 +344,9 @@ func cmdFromStruct(name string, dest path, t reflect.Type) (*command, error) {
 
 		// Look at the tag
 		var isSubcommand bool // tracks whether this field is a subcommand
-		for _, key := range strings.Split(tag, ",") {
+		kvPairs := strings.Split(tag, ",")
+
+		for _, key := range kvPairs {
```

Comment:
no need for this change given the suggestion above re using `spec.short == "" && spec.long == ""`

Reply:
:heavy_check_mark: